src/data_management/rasters/RasterLayerForm.tsx

```diff
@@ -183,10 +185,14 @@ const RasterLayerForm: React.FC<Props & PropsFromDispatch & RouteComponentProps>
       backUrl={"/data_management/rasters/layers"}
     >
       <form
-        className={formStyles.Form}
+        // remove this class because it gives element height
+        // className={formStyles.Form}
         onSubmit={handleSubmit}
         onReset={handleReset}
+        id={"raster_layer_form_id"}
       >
+      </form>
+      <div className={formStyles.Form}>
```

Comment:
I am not sure if it is a good choice here. Do I understand correctly that you made a form that will handle the onSubmit event for the real form container (which is now a div)? Is it because otherwise it will get conflict with the color map form? I think in general the decision to use a form for the custom color map modal (inside the raster layer form) is costing us a lot now. It can become simpler if it is just a modal component that will output a value based on the inputs.